swift-ticketing/src/swift_ticketing/db/booking.clj

```diff
@@ -24,7 +24,7 @@
                 :where [[:= :booking_id [:cast booking-id :uuid]]]})))
```

Comment:
Please add testing & linting to this repo as an action. Use Goose's workflow for reference: https://github.com/nilenso/goose/blob/main/.github/workflows/test_lint.yml